cPP_DSC.adoc

```diff
@@ -653,12 +649,12 @@ Extended SFRs (i.e. those SFRs that are not defined in [CC2]) are identified by
 
 FCS_CKM.1 Cryptographic Key Generation
 
-FCS_CKM.1.1:: The TSF shall generate cryptographic keys in accordance with a specified cryptographic key generation algorithm *corresponding to [.underline]#[selection:*#
+FCS_CKM.1.1:: The TSF shall generate cryptographic keys in accordance with a specified cryptographic key generation algorithm *corresponding to* [*selection*:
 +
-* [.underline]#*Asymmetric keys generated in accordance with FCS_CKM.1/AKG,*#
-* [.underline]#*Symmetric keys generated in accordance with FCS_CKM.1/SKG,*#
-* [.underline]#*Derived keys generated in accordance with FCS_CKM.5*#
-* [.underline]#*Derived keys generated in accordance with FCS_CKM_EXT.8*#
+* _Asymmetric keys generated in accordance with FCS_CKM.1/AKG,_
```

Comment:
Yes, that is correct. What we are doing here is not completing a selection, it is replacing an assignment with a selection (which kind-of completes the assignment, but also performs a refinement). There are very few places where we actually wholly/partially complete a selection (since that would be a selection defined in CC22/ECD).

Reply:
ok ,just wanted to make sure this is how we are going. I don't have an issue with it, but wanted to be certain we agreed.